util/archutil/amd64_binary.go

```diff
@@ -3,7 +3,7 @@
+const Binaryamd64 = "\x1f\x8b\x08\x00\x00\x00\x00\x00\x02\xff\xec\x98\x3f\x8b\x13\x41\x18\xc6\x9f\xd9\x6c\xee\x0f\x08\xb9\xe0\x81\xc2\x59\xa8\x58\xd8\xb8\xe1\x4e\x41\x05\x95\x55\x38\x1d\x45\x6d\xe4\x2c\x5d\x36\xd9\xe5\x5c\xf0\x72\xcb\xed\xec\x11\xe1\xe0\x2e\xe4\x3b\x88\x75\x40\xb1\xb0\x14\x4c\x99\xca\xc5\xd2\xc6\xda\x14\x42\xd0\xca\xce\x80\xa2\xec\xe6\x9d\x24\x3b\x26\x2a\xd8\xc9\xfc\x20\x3c\xf3\x3e\xfb\xce\xbc\x43\x92\x66\x9f\xfd\xf5\xdb\xd7\x0d\xc6\x20\x31\x70\x05\x59\xb5\x64\x67\xb5\x4d\xfe\x87\xf2\xa8\x05\x36\x2e\xa0\x00\x1b\x73\x28\x66\xbd\x26\x26\xb1\x73\x7a\x88\x8e\x96\x8a\xa5\xa1\xa4\x65\x71\xa2\x96\xf3\xa4\x86\x64\x4b\x95\x7d\x26\x7d\xfa\x64\xf7\x69\x8e\xd4\x53\xe4\x4b\x35\x27\x74\x19\x40\x01\xc0\x8d\xbb\x1b\xf0\x9f\xbd\xbe\xf8\xfe\xc5\xc7\xfb\x77\xbe\x3f\x39\xfc\xed\xf9\xa7\xfd\x37\x67\xdf\xbd\x82\x46\xa3\xd1\x68\x34\x1a\x8d\x46\xa3\xd1\x68\x34\xff\x29\x7c\xb5\x5b\x6a\xf3\xd6\xd7\xf9\xbd\x5b\xbc\x03\xe0\x20\x35\x4b\xed\xcb\x0c\x38\xd8\x3b\xcf\x93\x2e\xcb\x9a\xde\x96\xda\xbc\xd9\x63\x27\x9f\x82\x37\x07\xa9\xc4\x2b\xbc\xc3\xa8\x7d\xf8\xb8\xd5\x63\xbc\x35\x60\xb1\xf1\x79\x91\x27\xc9\x55\x00\xd9\xe2\x5a\xba\x00\x4f\xba\x97\xd2\x83\x8b\xbf\xbb\x4b\x01\x6c\xf4\x1e\x9f\xf7\x8d\x71\x3e\x80\x71\x7e\x60\xe2\xcb\x0f\xb5\xb7\x4c\x29\xc6\x86\xd2\xbf\x4c\xfe\x03\xc5\x3f\x42\xfe\xa6\xe2\x9f\xc8\x22\x87\x5f\xe7\x1e\x93\xfe\xf1\xbc\x7f\x7a\x86\x5f\x99\xe1\xa3\x22\xb6\xc2\x4a\xad\xb6\xb6\x1e\xd4\xee\xdd\xb4\xb6\xb1\xbb\x8a\xdd\x35\xf8\x8d\x40\xc0\x71\xaa\x51\xe4\x44\xc2\xdd\x11\x70\x7c\xcf\x15\x2e\x1c\xbf\xee\x01\x56\xf4\x78\x4b\xb8\x55\x58\x91\xd8\x19\xea\x43\xb9\xaa\x6f\x0b\xdf\xda\xac\xc7\x56\x35\x0e\x1e\x79\x67\x02\x0f\x96\xf0\x1b\xe2\x9f\xff\x1f\x2b\x00\xe6\xb3\x6f\x48\xcd\x5b\xf2\x39\x0b\x94\xbc\x45\x62\xd1\x6f\x35\x37\xca\x71\xec\x5c\x9e\x13\x2a\xfd\x6c\x4a\x6d\x4c\xb9\x57\x48\xfb\x17\xd8\x78\x6e\x7a\xcf\x05\x7a\x7e\x94\x74\x91\x32\x1f\x95\x06\xe5\x59\xe7\xfe\x30\xbf\x3c\x63\xff\xcb\xbf\xdc\xff\x33\x00\x00\xff\xff\xad\x50\x19\xd5\x60\x13\x00\x00"
```

Comment:
This diff looks odd when generating bins locally: https://github.com/moby/buildkit/actions/runs/7740278596/job/21104904747#step:4:682

Reply:
Ok when reverting this change I still got a diff on ci: https://github.com/moby/buildkit/actions/runs/7740278596/job/21104904747#step:4:682

Something is flaky, I will add an exception in the meantime.

Reply:
Added an exception in validation: https://github.com/moby/buildkit/pull/4607/commits/aa1457345463ef9f41216e03362acac931575ce1